pkg/trace/agent/agent.go

```diff
@@ -218,6 +219,7 @@ func (a *Agent) loop() {
 	} {
 		stopper.Stop()
 	}
+	timing.Stop()
```

Comment:
Do we want to defer this instead?

Reply:
Can do! I put here in the `loop()` method near the stoppers for consistency but can use a defer in `Run` instead since `Run` is blocking.